### PR TITLE
[CGKIELE-385] Compile contracts

### PR DESCRIPTION
**Pull request by rtkaczyk**

### CODE REVIEW

File: lib/mallet.js [hunk 1]
```diff
@@ -124,6 +125,120 @@ class Iele {
...
+      const bytecode = this.compileIele(code).result;
+      result.bytecode = bytecode;
+      return result;
+    } else {
+      result.error = true;
+      return result;
+    }
+  }
+
+  compileIele(source) {
```
Comment (LukasGasior1) [inline lib/mallet.js]: this method is quite similar in structure to compileSol. Can we extract & reuse some code?

File: lib/mallet.js [hunk 2]
```diff
@@ -124,6 +125,120 @@ class Iele {
     tx.data = '0x' + rlp.encode([code, args]).toString('hex');
     return this.mallet.sendTransaction(tx, password);
   }
+
+  compileSol(source) {
```
Comment (LukasGasior1) [inline lib/mallet.js]: maybe `compileSolToIele` to make it clear what it does?

Comment (rtkaczyk) [reply lib/mallet.js]: Well, it is part of `Iele` class, besides it's not really meant to be called by users. I will add `_` to those names.

Comment (LukasGasior1) [reply lib/mallet.js]: ok, nvm I didn't notice it's inside Iele class, I thought it's directly under mallet.js file

**Pull request merged**